docs/5. Integrations/Filesearch Using OpenAI Assistants.md

```
@@ -47,7 +47,7 @@ Define the following parameters:
 
 
 - Provide a system prompt in the `Instructions` field.  
-  *[Click Here](https://glific.org/a-simple-guide-to-using-large-language-models/#prompt) to read more on prompt engineering.*
+  *[Click Here](https://glific.org/a-simple-guide-to-using-large-language-models/#prompt) to read more on prompt engineering, or see [Writing effective instructions](https://glific.github.io/docs/docs/Integrations/Creating%20and%20modifying%20assistants%20in%20Glific/#writing-effective-instructions) for a prompt template and examples.*
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)